js/helpers.mjs

```diff
@@ -1,43 +1,63 @@
-
 const pageContainerEl = document.querySelector('.page');
 const textareaEl = document.querySelector('.page > .textarea');
 const overlayEl = document.querySelector('.page > .overlay');
 
+export const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
 
-const isMobile = /iPhone|iPad|iPod|Android/i.test(navigator.userAgent);
-
-function applyPaperStyles() {
+export const applyPaperStyles = () => {
   pageContainerEl.style.border = 'none';
   pageContainerEl.style.background = 'linear-gradient(to right,#eee, #ddd)';
-  overlayEl.style.background = `linear-gradient(${Math.random()*360}deg, #0008, #0000)`
+  overlayEl.style.background = `linear-gradient(${
+    Math.random() * 360
+  }deg, #0008, #0000)`;
   overlayEl.style.display = 'block';
   textareaEl.classList.add('paper');
-}
+};
 
-function removePaperStyles() {
+export const removePaperStyles = () => {
   pageContainerEl.style.border = '1px solid #ccc';
   pageContainerEl.style.background = 'linear-gradient(to right,#fff, #fff)';
   overlayEl.style.display = 'none';
   textareaEl.classList.remove('paper');
-}
+};
 
-function addFontFromFile(fileObj) {
+export const addFontFromFile = (fileObj) => {
   const reader = new FileReader();
-  reader.onload = e => {
+  reader.onload = (e) => {
     const newFont = new FontFace('temp-font', e.target.result);
-    newFont.load()
-      .then(loadedFace => {
-        document.fonts.add(loadedFace);
-        textareaEl.style.fontFamily = 'temp-font';
-      })
-  }
-  reader.readAsArrayBuffer(fileObj)
-}
-
-
-export {
-  isMobile,
-  applyPaperStyles,
-  removePaperStyles,
-  addFontFromFile
-}
\ No newline at end of file
+    newFont.load().then((loadedFace) => {
+      document.fonts.add(loadedFace);
+      textareaEl.style.fontFamily = 'temp-font';
+    });
+  };
+  reader.readAsArrayBuffer(fileObj);
+};
+
+/**
+ *
+ * @param {string} hashval
+ */
+export const smoothlyScrollTo = (hashval) => {
+  let target = document.querySelector(hashval);
+  target.scrollIntoView({
+    behavior: 'smooth',
+    block: 'start',
+  });
+  history.pushState(null, null, hashval);
+};
+
+/**
+ *
+ * @param {string} _
+ */
+export const $ = (_) => document.querySelector(_);
+
+/**
+ *
+ * @param {string} selector
+ * @param {string} eventName
+ * @param {function} callback
+ */
+export const addEvent = (selector, eventName, callback) => {
+  document.querySelector(selector).addEventListener(eventName, callback);
+};
```

Comment:
Won't this make code a little unfamiliar for new people?

 It is ok if the code gets large or we have to type more words but these wrappers can make code complicated for a new person. 

Let's say I am a new person who wants to contribute I come and see a function called `addEvent()` seeing this I won't necessarily know its a wrapper so I will have to go and look up the helper function as well. Whereas if it is `addEventListener` right away I would know what it does and how to use it.

Also if I want `element.querySelector` instead of `document` (though it is a rare case) or pass the third parameter to the function, I will either have to update this function and make it more complicated or add event listener the usual way.

Reply:
ok i'll revert it